charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -527,48 +531,6 @@ collectors:
               - batch
             exporters:
               - debug
-  cluster:
```

Comment:
do you think users would want this to be configurable? i.e. we give the user the ability to choose if they want the standalone or use leader election? I think there's value in having the option, but this is meant to be an opinionated chart... my only hesitation is that using leader election can be a bit frustrating for some users architectures.

Reply:
I've added an example about how to define back the previous approach as well as some notes in readme.md to explain how to upgrade and to announce the "new requirement" of leader election extension and linking the example that defines back the previous approach.

To disable the leader election I have had to add a new field to the preset because I haven't found any other good way to disable it without adding a lot of code (but if someone has a good idea that I've missed just say them to me, I'm quite newbie with advanced helm templating 😄 )

Reply:
@JorTurFer your implementation is fantastic in my eyes :) thanks for the work here! I like how this ended up.